src/Components/OATHeader/OATHeader.tsx

```diff
@@ -77,6 +91,42 @@ const OATHeader = ({ elements, onImportClick }: OATHeaderProps) => {
         }
     ];
 
+    useEffect(() => {
+        const newFiles = [];
+        acceptedFiles.forEach((sF) => {
+            if (sF.type === 'application/json') {
+                newFiles.push(sF);
+            }
```

Comment:
How do we inform the user about files that we don't accept?

Reply:
Then Again using The Error Handling Component should be a good Idea for this Also

Reply:
Let's use an alert until then.